crates/bitcoin-da/src/service.rs

```diff
@@ -56,6 +56,9 @@ use crate::REVEAL_OUTPUT_AMOUNT;
 pub const FINALITY_DEPTH: u64 = 8; // blocks
 const POLLING_INTERVAL: u64 = 10; // seconds
 
+const MEMPOOL_SPACE_RECOMMENDED_FEE_ENDPOINT: &str =
+    "https://mempool.space/testnet4/api/v1/fees/recommended";
+
```

Comment:
IMO this should be set via config and it should be optional. If the url is not passed in config, then estimate_smart_fee should still be used

Reply:
talked with @eyusufatik and decided it was not necessary, check the new impl after last commit